Files/Filesystem/FilesystemOperations/FilesystemOperations.cs

```diff
@@ -575,7 +577,7 @@ public async Task<IStorageHistory> CreateAsync(IStorageItemWithPath source, IPro
             if (fsResult == FileSystemStatusCode.Unauthorized)
             {
                 // Try again with fulltrust process
-                await associatedInstance.ServiceConnection?.Elevate();
+                //await associatedInstance.ServiceConnection?.Elevate();
```

Comment:
What changed here?

Reply:
Disabled the request for elevating the fulltrust process. Feature needs a little more work (e.g there's no warning/indication that you're going to delete the file as admin)